tests/sentry/runner/commands/badpatch.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Why do we use configmaps? Configmaps are config files for kubernetes resource, since we are passing the options file to a script, this does not need to be a configmap. 

The original idea of configmap was to mount it to getsentry and load options directly from there. Since we go through the k8s job and the script created here, this does not need to be a configmap. It can be just a simple yaml file without all the boilerplate:

```
fetch : ["github-login.api-domain"]
update :
  github-login.api-domain  : [333, 22, 1]
  nodedata.cache-sample-rate : 4.0
delete : ["relay.transaction-metrics-org-sample-rate"]
```